app/iht/views/application/declaration/probate_application_form_details.scala.html

```diff
@@ -0,0 +1,46 @@
+@import iht.models._
+@import iht.models.application.ProbateDetails
+
+@import iht.views.html._
+@import iht.utils._
+@import iht.constants.IhtProperties
+@import uk.gov.hmrc.play.partials.FormPartialRetriever
+
+@(probateDetails: Option[ProbateDetails],
+        registrationDetails: RegistrationDetails)(implicit request:Request[_], messages: Messages, ihtFormPartialRetriever: FormPartialRetriever)
+
+@iht_main_template_application(title = Messages("page.iht.application.probate.title"),
+    browserTitle = Some(Messages("page.iht.application.probate.browserTitle"))
+){
+    <p>@Messages("page.iht.application.probate.guidance.p1", DeceasedInfoHelper.getDeceasedNameOrDefaultString(registrationDetails))</p>
+
+    <table>
+        <caption class="visually-hidden">@Messages("page.iht.application.probate.data.visuallyHiddenCaption", DeceasedInfoHelper.getDeceasedNameOrDefaultString(registrationDetails))</caption>
+        <tbody>
+            <tr>
```

Comment:
seven_years_gift_values.scala.html has some css attributes against table rows and other table elements - e.g. table__row--responsive: we probably need to check with Adam what these are for and whether we need them on this new table.

Reply:
I think because of the simplicity of this table, and the absence of styling on the columns/rows, it should be responsive without any additional classes.